frontend/src/hooks/use-mutate.js

```diff
@@ -48,9 +48,31 @@ export const useMutate = () => {
       }
     })
 
+    const addTag = useMutation((input) => gqlHelper(ADD_NEW_TAG, input),
+    {
+      onSuccess: (response) => {
+        checkResponse(response)
+      },
+      onError: (e) => {
+        sendError(e.response.data.errors[0].message)
+      }
+    })
+
+    const updateTag = useMutation((input) => gqlHelper(UPDATE_TAG, input),
+    {
+      onSuccess: (response) => {
+        checkResponse(response)
```

Comment:
Similar to the above comment, since we're updating a single product, we need to invalidate both the single tag in react-query along with all-tags.

Reply:
One solution would be to add an additional `...args` to checkResponse and pass in the invalidation strings in there to loop through and invalidate the react-queries